pubspec.yaml

```diff
@@ -14,6 +14,8 @@ dependencies:
   bordered_text: ^1.0.0
   get_it: ^4.0.4
   mockito: ^4.1.1
+  firebase_core:
+  cloud_firestore:
```

Comment:
Define specific dependency number, please.

Reply:
Its not necessary here, pub get takes the latest one automatically